specs/swagger.yml

```diff
@@ -3083,6 +3083,7 @@ responses:
     schema:
       $ref: '#/definitions/createEnvelope'
   sessionCreateResponse:
+    description: The create request was successful and the resource has been added at the ollowing location.
```

Comment:
is this really only an 'add' and not and 'add/delete'? pls look at this i feel like this is a mistake with `ollowing`

Reply:
the line below seems to be overriding this one?

Reply:
Looks like an auto merge issue. Removed the extra line.